pkg/sync/file/filepath_sync.go

```diff
@@ -17,8 +17,8 @@ import (
 )
 
 type Sync struct {
-	URI          string
-	Source       string
+	URI string
```

Comment:
It looks like this section needs to be formatted.

Reply:
done